torch/linalg/__init__.py

```diff
@@ -431,6 +431,45 @@
             [1, 2, 2, 2]])
 """)
 
+multi_dot = _add_docstr(_linalg.linalg_multi_dot, r"""
+linalg.multi_dot(tensors, *, out=None)
```

Comment:
Reviewer's note: the use of "tensors" here is consistent with the language in other functions that accept a list of tensors, like torch.cat (https://pytorch.org/docs/master/generated/torch.cat.html?highlight=cat#torch.cat)